docs/getting-started.md

```diff
@@ -264,3 +264,25 @@ kubectl logs -f -n aws-provider-system aws-provider-controller-manager-0
 [kubectl]: https://kubernetes.io/docs/tasks/tools/install-kubectl/
 [aws-vault]: https://github.com/99designs/aws-vault
 [kustomize]: https://github.com/kubernetes-sigs/kustomize
+
+### Target cluster's control plane machine is up but target cluster's apiserver not working as expected
+
+If `aws-provider-controller-manager-0` logs did not help, you might want to look into cloud-init logs, /var/log/cloud-init-output.log, on the controller host.
+Also, verifying kubelet status and logs may also provide hints:
+```bash
+journalctl -u kubelet.service
+systemctl status kubelet
+```
+For reaching controller host:
+
+* Find out public IP of the bastion node. 
+* copy private key(or the .pem), corresponding to the key-pair discussed in the `ssh key pair` section above, to the bastion node:
```

Comment:
Torn between saying to do this vs. using SSH agent forwarding with `ssh -A <node>`

Reply:
@randomvariable updated. PTAL!